index.html

```diff
@@ -12,7 +12,7 @@
 <body class="navbar">
     <header>
         <ul>
-            <li>About</li>
+            <li>Abouts</li>
```

Comment:
Issues In Html ♻️
please kindly remove the class name of the navbar from the body since the body can't be a navbar
please kindly also remove the class name nav from the section since it's not the proper naming convention for the section
please kindly try to keep one heading tag per page since it's better for SEO
please kindly use as the templete

Reply:
Well done with these good review comments. 

- [ ] However, as per the requirement you are expected to make at least two in-line comments.  But you have four-in-one comments and as such did not highlight the part of the code for each comment. Always highlight the section of the code that your comment concerns. So you would have had about four separate in-line comments. 
- [ ] Your first two comments were okay in terms of **_Politeness_**,  **_clarity_**, _**why**_ and _**what**._ . The **_why_** speaks to why you are suggesting the changes. **_What_** speaks to what should be done or what the issue is. **_Clarity_** speaks to how it is easy to understand without ambiguity. Politeness speaks about the manner in which the issue is communicated. 
- [ ] If you have more than one thing to say about an issue, please use bullet points for ease of assimilation. 

Please always ensure your comments point to these - What, why, Politeness and clarity. Please update the last two comments to reflect these points. **_Remember to keep them as separate in-line comments._**

Reply:
> Issues In Html ♻️ please kindly remove the class name of the navbar from the body since the body can't be a navbar please kindly try to keep one heading tag per page since it's better for SEO

- [ ] As the previous reviewer suggested 👍 , kindly ensure that you have atleast 2 inline comments. In addition, you should try as much as possible to provide detailed explanation and ensure that student clearly understands your request 👌 .
- [ ] [OPTIONAL] Lets try as much as possible to use `gifs` and [imojis](https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md) 🥰 . This helps the student to have a friendly feeling when reading your comment ☺️